src/runtime/config/configuration-remote.toml.in

```diff
@@ -0,0 +1,299 @@
+# Copyright (c) 2017-2019 Intel Corporation
+# Copyright (c) 2023 IBM Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+# XXX: WARNING: this file is auto-generated.
+# XXX:
+# XXX: Source file: "@CONFIG_REMOTE_IN@"
+# XXX: Project:
+# XXX:   Name: @PROJECT_NAME@
+# XXX:   Type: @PROJECT_TYPE@
+
+
+[hypervisor.remote]
+remote_hypervisor_socket = "/run/peerpod/hypervisor.sock"
+remote_hypervisor_timeout = 600
+
+
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one, returning an error if none is
+# available, or none is supported by the hypervisor.
+#
+# Known limitations:
+# * Does not work by design:
+#   - CPU Hotplug 
+#   - Memory Hotplug
+#   - NVDIMM devices
+#
+# Default false
+# confidential_guest = true
+
+
+# List of valid annotation names for the hypervisor
+# Each member of the list is a regular expression, which is the base name
+# of the annotation, e.g. "path" for io.katacontainers.config.hypervisor.path"
+enable_annotations = @DEFENABLEANNOTATIONS@
```

Comment:
remote hypervisor is not handling any annotations. Should this be commented out or set to empty string ?

Reply:
@bpradipt What about other things that you do differently, e.g. instance sizes? Should we completely "rewrite" the comments around `default_memory` or `default_vcpus`?

What about just keeping the original commit as a "clone" of what is done for other cases, and then adding another commit that makes the differences clear? Maybe you could contribute that second commit on top of @stevenhorsman's initial commit? That way, it would make it quite clear what was changed specifically for the remote hypervisor case.

Reply:
Good point, I'll just update this to the empty array

Reply:
Sorry, that previous commit was to Pradipta's suggestion - when I wrote it your comment wasn't showing up for my Christophe. I think your comment is interesting though. When creating this I was struggling to work out a balance between hard-coded preferences, commenting out defaults that we'd just ignore, or deleting options altogether, so I think understanding the least-confusing approach for the user will be important here and maybe a combination of hard-coded defaults, or commented out values with an explanation about the remote-hypervisor not using them would be best, so I'll have a go at that and push an update. Thanks!

Reply:
> combination of hard-coded defaults, or commented out values with an explanation about the remote-hypervisor
I think this is a good approach. Will make it easier for the admin to identify what parameters are not used for remote.